src/Razor/src/Microsoft.AspNetCore.Razor.OmniSharpPlugin.StrongNamed/OmniSharpTelemetryReporter.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Text;
+using Microsoft.AspNetCore.Razor.Common.Telemetry;
+using Microsoft.VisualStudio.Telemetry;
+
+namespace Microsoft.AspNetCore.Razor.OmniSharpPlugin.StrongNamed
+{
+    internal class OmniSharpTelemetryReporter : ITelemetryReporter
```

Comment:
Can we not export this, so things still use MEF? Presumably eventually we'll want something in here

Reply:
I tried the MEF-export approach but the class wasn't getting imported properly, not sure why. I agree we should ideally still use MEF. However, since the class is non-functional at the moment and in the interest of getting the VSCode world unblocked, I feel we can just leave it for now.